CMakeLists.txt

```diff
@@ -361,6 +356,14 @@ if(pyAMReX_BUILD_TESTING)
 endif()
 
 
+# Warnings ####################################################################
+#
+# FIXME _ALL_TARGETS is not defined
+foreach(pyamrex_tgt IN LISTS _ALL_TARGETS)
+    pyamrex_set_compile_warnings(${pyamrex_tgt})
+endforeach()
```

Comment:
This does not work, as is. Need to understand how to loop over all targets in this case.

Reply:
Fixed :)